### PR TITLE
OSDOCS#8727:QE and Style Revisions of MOBB Content for Using AWS controllers for Kubernetes (ACK) on ROSA

### PR DESCRIPTION
**Pull request by AedinC**
This is a style revision and full QE review for the Using AWS controllers for Kubernetes (ACK) on ROSA cloud experts tutorial.

Version(s):
4.14+

Issue:
[OSDOCS-8727](https://issues.redhat.com/browse/OSDOCS-8727)

Link to docs preview:
[Tutorial: Using AWS Controllers for Kubernetes on ROSA](https://67839--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-using-aws-ack)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc [hunk 1]
```diff
@@ -18,22 +18,24 @@ toc::[]
...
+link:https://aws-controllers-k8s.github.io/community/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from {product-title} (ROSA). With ACK, you can take advantage of AWS-managed services for your applications without needing to define resources outside of the cluster or run services that provide supporting capabilities such as databases or message queues within the cluster.
 
-Users can install various ACK Operators directly from OperatorHub. This makes it relatively easy to get started and start using it with your applications. This controller is a component of the AWS Controller for Kubernetes project. This project is currently in developer preview.
+You can install various ACK Operators directly from OperatorHub. This makes it easy to get started and use the Operators with your applications. This controller is a component of the AWS Controller for Kubernetes project, which is currently in developer preview.
```
Comment (mletalie) [inline cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc:23]: Question (based on my limited knowledge of the relation between operators/controllers): Is this statement saying the specific controller is in developer review or the project itself (which this controller is a part of)? I guess the safe assumption would be the controller is also in developer preview, b/c it is part of the overall project? I guess my other question would be what is the difference between tech preview (which I believe is a term we often use) and "developer preview"? May not be related to this doc, just wondering if they are interchangeable.

File: cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc [hunk 2]
```diff
@@ -18,22 +18,24 @@ toc::[]
...
 
+[id="cloud-experts-using-aws-ack-prerequisites"]
 == Prerequisites
 
 * A ROSA cluster
 * A user account with `cluster-admin` privileges
-* You have access to the OpenShift CLI (`oc`)
-* You have access to the AWS CLI (`aws`)
+* The OpenShift CLI (`oc`)
+* The Amazon Web Services (AWS) CLI (`aws`)
```
Comment (mletalie) [inline cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc:34]: No rosa CLI prereq? I guess the oc CLI will suffice, just wondering why we wouldn't push/suggest the ROSA CLI given it is a ROSA based tutorial? FWIW, I have seen a lot of oc CLI (vs rosa CLI) prereqs in these tutorials so I guess that must be the standard prereq.

**Review by mletalie — COMMENTED** [review]
/label peer-review-done

**Comment by mletalie** [comment]
Just  a couple of questions/comments, otherwise LGTM!

File: cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc (hunk 2 above)
Comment (AedinC) [reply cloud_experts_tutorials/cloud-experts-using-aws-ack.adoc:34]: Yeah it's a valid question. I guess it's cos this tutorial has commands that are either in the `AWS` or `oc` CLIs.

**Pull request merged**